docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -0,0 +1,219 @@
+
+## Security setup
+
+This installation will not succeed if you have enabled the [PodSecurityPolicy](*https://v1-23.docs.kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy) admission controller or if you are enforcing the Restricted policy with [Pod Security](https://v1-24.docs.kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces) admission controller. The reason is that the installation includes a deployment of MinIO. The [minio/minio chart](https://github.com/minio/minio/tree/master/helm/minio) is not compatible with running under a Restricted policy or the PodSecurityPolicy that the mimir-distributed chart provides.
+
+If you are using the PodSecurityPolicy admission controller, then it is not possible to deploy the mimir-distributed chart with MinIO. Refer to Run Grafana Mimir in production using the Helm chart for instructions on setting up an external object storage and disable the built-in MinIO deployment with minio.enabled: false in the Helm values file.
+
+If you are using the Pod Security admission controller, then MinIO and the tempo-distributed chart can successfully deploy under the baseline pod security level.
+-->
+
+## Install the Helm chart in a custom namespace
+
+Using a custom namespace solves problems later on because you do not have to overwrite the default namespace.
+
+1. Create a unique Kubernetes namespace, for example tempo-test:
+
+```bash
+kubectl create namespace tempo-test
+```
+
+1. For more details, see the Kubernetes documentation about [Creating a new namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces/#creating-a-new-namespace).
+
+1. Set up a Helm repository using the following commands:
+
+   ```bash
+   helm repo add grafana https://grafana.github.io/helm-charts
+   helm repo update
+   ```
+
+   >**NOTE**: The Helm chart at https://grafana.github.io/helm-charts is a publication of the source code at grafana/tempo.
+
+### Set Google Cloud Platform as the storage option
+
+Before you run the Helm chart, you need to configure where trace data will be stored. These steps create a YAML file with the storage configuration.
+
+You need the `secret_key` from your service provider used as part of the user or account authentication. In this case, it refers to S3’s secret key. Google Cloud Storage has a similar option.
+
+1. Create a YAML file of Helm values called `custom.yaml`.
+
+1. Add the following configuration to the file to set Google Cloud Platform as the trace storage.
+   - Change `temorockstracing` to your `secret_key`.
+   - Change `bucket` to match the bucket name in GCP.
+   - Change `endpoint` to the correct one for your set up.
+
+   ```yaml
+   ---
+   minio:
+     enabled: false
+   storage:
+     trace:
+       backend: s3
+       s3:
+         access_key: 'tempo-test'
+         secret_key: 'temporockstracing'
+         bucket: 'tempo-test'
+         endpoint: 'stodev0.znet:9000'
```

Comment:
Endpoint will only be necessary I think in the case of minio, or a custom s3 endpoint.